### PR TITLE
Add outpatient register and referral note forms

### PR DESCRIPTION
**Pull request by Irenyak1**
Add s outpatient register and referral note forms for the OSEQ - UgandaEMR track challenge.

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -17,3 +17,7 @@ export const CHILD_REGISTER_ENCOUNTER_TYPE = "";
 //maternity
 export const ADMISSION_DATE = "cb9b223e-a09c-4f8a-b34d-a0934a2af04a";
 export const DELIVERY_TYPE = "dcc3ac63-30ab-102d-86b0-7a5022ba4115";
+
+// opd
+export const OUTPATIENT_REGISTER_ENCOUNTER_TYPE =
```
Comment (solemabrothers) [inline src/constants.ts:22]: @Irenyak1 please rename this OUTPATIENT_DEPARTMENT_ENCOUNTER_TYPE. In a facility setting, there is not encounter type known as Outpatient register encounter type

File: src/constants.ts [hunk 2]
```diff
@@ -17,3 +17,9 @@ export const CHILD_REGISTER_ENCOUNTER_TYPE = "";
...
+  "ee4780f5-b5eb-423b-932f-00b5879df5ab";
+export const REFARRAL_NOTE_ENCOUNTER_TYPE =
```
Comment (solemabrothers) [inline src/constants.ts:24]: @Irenyak1 check the spelling of  referral, also where is this UUID coming from? it seems not to exist in out database

File: src/constants.ts (hunk 1 above)
Comment (Irenyak1) [reply src/constants.ts:22]: @solemabrothers , ok let me do that

File: src/constants.ts (hunk 2 above)
Comment (Irenyak1) [reply src/constants.ts:24]: @solemabrothers  sure let me fix the spelling. The UUID is a place holder since many UUIDs have not been provided to us yet.

Comment (solemabrothers) [reply src/constants.ts:24]: @Irenyak1 work with @Mwanje to get you this UUID, there is already an existing UUID for the referral encounter.

Comment (Irenyak1) [reply src/constants.ts]: @solemabrothers sure I am going to get to him. I am in a black out and my laptop's battery is running out.

Comment (Irenyak1) [reply src/constants.ts]: @solemabrothers  @Mwanje @slubwama  any more reviews on this?

**Comment by Mwanje** [comment]
@Irenyak1, can you resolve these conflicts

**Comment by Irenyak1** [comment]
hello @pirupius before I resolve the conflicts in dashboard.meta.tsx. I would like to know the reason for the changes here 

**`export const opdDashboardMeta = {
  slot: "opd-dashboard-slot",
  config: { columns: 1, type: "grid" },
  title: "Outpatient Department",
};`**

as opposed to the former 

**`export const opdDashboardMeta = {
  title: "OPD",
  slotName: "opd-dashboard-slot",
};`**

because I relied on the **slotName: "opd-dashboard-slot"**  to create the 

**`export const outpatientDashboardMeta = {
  title: "Outpatient Register",
  slot: "outpatient-dashboard-slot1",
  config: { columns: 1, type: "grid" },
};`**

and 

**`export const referralNoteDashboardMeta = {
  title: "Referral Note",
  slot: "referral-note-dashboard-slot",
  config: { columns: 1, type: "grid" },
};`**

which has been holding the out patient register and the referral note forms.  Could we have a call today or early next week in case you need me to explain more?

**Comment by pirupius** [comment]
Hey @Irenyak1 the slot name `opd-dashboard-slot` remained the same and the only change i made for that specific meta was adding the config.

**Comment by Irenyak1** [comment]
@pirupius  sure I saw that, but when you add the config and change slotName to slot, that ceases to be a section for creating Dashboard Group you can only create  Dashboard Link. Though in this case I would think that we want to group the referral note and opd register under a group called OPD. How do you see about that?

**Comment by pirupius** [comment]
@Irenyak1 i understand what you mean. This PR hadn't been merged in yet so i didn't know the plan for this. You should maintain the `slotName`.

**Review by pirupius — APPROVED** [review]
Any pending issues will be resolved after the packaging PR is merged

**Pull request merged**